src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.form

```diff
@@ -100,35 +99,31 @@
         <Property name="font" type="java.awt.Font" editor="org.netbeans.beaninfo.editors.FontEditor">
           <Font name="Segoe UI" size="18" style="0"/>
         </Property>
-        <Property name="text" type="java.lang.String" value="Custom Lookup"/>
+        <Property name="text" type="java.lang.String" value="Custom lookup"/>
```

Comment:
Did you mean to rename the text of this label?

Reply:
Yes

Reply:
Why is that? It doesn't seem related to the title of the PR?